package.json

```diff
@@ -28,8 +28,10 @@
     "author": "Plotly <chris@plot.ly>",
     "license": "MIT",
     "dependencies": {
-        "@material-ui/core": "^4.11.4",
-        "@material-ui/icons": "^4.11.3",
+        "@mui/material": "^5.12.1",
+        "@mui/icons-material": "^5.11.16",
+        "@emotion/styled": "^11.10.6",
+        "@emotion/react": "^11.10.6",
```

Comment:
Why are these `@emotion` packages here?

Reply:
Dependency of `@mui/material`

Reply:
I see, @BSd3v pointed me to this odd aspect of their [installation instructions](https://mui.com/material-ui/getting-started/installation/) that you need to explicitly install those even if you don't use them directly 🙄

Reply:
Is there any way to just include the row menu component rather than the entire package?